docs/_templates/indexcontent.html

```diff
@@ -0,0 +1,47 @@
+{% extends "defindex.html" %}
+
+{% block body %}
+
+<div class="figure align-center" id="id1">
+    <img src="_images/logo.svg">
+    <h1>Kinto — Store, Sync, Share and Self-Host.</h1>
```

Comment:
For a people that love to put commas *everywhere*, it's odd that you'd be against one that actually makes grammatical sense. :wink: